system/services/assetManager/AssetManagerService.cfc

```diff
@@ -936,6 +936,26 @@ component {
 		return settings;
 	}
 
+	public any function listEditorDerivates(){
+		var derivatives = _getConfiguredDerivatives();
+		var publicDerivatives = ["none"];
+
+		for( var derivative in derivatives ) {
+
+			if ( derivatives[ derivative ].keyExists( "inEditor" ) ) {
+
```

Comment:
There's a lot of extraneous whitespace here. would prefer there not to be all these extra empty lines.